### DISCUSSION TITLE
Villager bones/headstones

### DISCUSSION THREAD

**Discussion by GhostOpsX** [Luke100000/minecraft-comes-alive#630] [Q&A] [answered]
Hi, I accidently broke one of my villagers headstones and it dropped his bones, is there any way I can create a new headstone for him or is there anything I can do with his bones to create a new one? iv tried creating a new headstone with his bone as a replacement for normal one you'd use and it didn't change anything? any help would be appreciated, thanks!

**Comment by Luke100000** [comment, marked as answer]
No, the soul has been released into the Aether, but I'm working on placing a new gravestone (even if you cant revive anymore)
